diff --git a/src/sysclip.cc b/src/sysclip.cc
index 4b4830d..5ebca2f 100644
--- a/src/sysclip.cc
+++ b/src/sysclip.cc
@@ -69,8 +69,8 @@ static clip_command *get_commands()
             { { nullptr, nullptr } },
     };
     static clip_command XCLIP_CMDS[] = {
-            { { "xclip -i > /dev/null 2>&1",
-                    "xclip -o < /dev/null 2>/dev/null" } },
+            { { "xclip -i -selection clipboard > /dev/null 2>&1",
+                    "xclip -o -selection clipboard < /dev/null 2>/dev/null" } },
             { { nullptr, nullptr } },
     };
     static clip_command XSEL_CMDS[] = {
